dev/import-beats/packages.go

```diff
@@ -324,7 +324,7 @@ func (r *packageRepository) save(outputDir string) error {
 
 			// dataset/elasticsearch
 			if len(dataset.elasticsearch.ingestPipelines) > 0 {
-				ingestPipelinesPath := filepath.Join(datasetPath, "elasticsearch", "ingest-pipeline")
+				ingestPipelinesPath := filepath.Join(datasetPath, "elasticsearch", util.DIR_INGEST_PIPELINE)
```

Comment:
is this correct path (caps)?

Reply:
:-( No. Seems like my IDE did not catch this during refactoring :-(